From: Stephane Glondu <steph@glondu.net>
Date: Sun, 18 Apr 2010 15:12:10 +0200
Subject: [PATCH] Set version number

---
 VERSION |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/VERSION b/VERSION
index 9d186bd..1e53d7b 100644
--- a/VERSION
+++ b/VERSION
@@ -1,4 +1,4 @@
-3.12.0+dev17 (2010-01-20)
+3.12.0+dev17+10268
 
 # The version string is the first line of this file.
 # It must be in the format described in stdlib/sys.mli
-- 
