From: Stephane Glondu <steph@glondu.net>
Date: Sun, 18 Apr 2010 15:12:10 +0200
Subject: [PATCH] Set version number

---
 VERSION |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/VERSION b/VERSION
index 58fc0bd..af2edec 100644
--- a/VERSION
+++ b/VERSION
@@ -1,4 +1,4 @@
-3.12.0+dev23 (2010-05-12)
+3.12.0+dev23+10396
 
 # The version string is the first line of this file.
 # It must be in the format described in stdlib/sys.mli
-- 
