From: Stephane Glondu <steph@glondu.net>
Date: Wed, 20 Jun 2012 21:58:35 +0200
Subject: ocamlopt fix on powerpc

Origin: http://caml.inria.fr/cgi-bin/viewvc.cgi?view=revision&revision=12583
---
 asmcomp/power/arch.ml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/asmcomp/power/arch.ml b/asmcomp/power/arch.ml
index 696073e..239390c 100644
--- a/asmcomp/power/arch.ml
+++ b/asmcomp/power/arch.ml
@@ -48,7 +48,7 @@ let size_float = 8
 
 (* Behavior of division *)
 
-let division_crashes_on_overflow = false
+let division_crashes_on_overflow = true
 
 (* Operations on addressing modes *)
 
-- 
