From: Stephane Glondu <steph@glondu.net>
Date: Wed, 20 Jun 2012 22:00:46 +0200
Subject: Move power.o rule in asmcomp test after definition of $(SYSTEM)

---
 testsuite/tests/asmcomp/Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/testsuite/tests/asmcomp/Makefile b/testsuite/tests/asmcomp/Makefile
index fb1af49..c4d1aa0 100644
--- a/testsuite/tests/asmcomp/Makefile
+++ b/testsuite/tests/asmcomp/Makefile
@@ -141,11 +141,11 @@ clean: defaultclean
 	@rm -f parsecmm.ml parsecmm.mli lexcmm.ml
 	@rm -f $(CASES:=.s)
 
-power.o: power-$(SYSTEM).o
-	@cp power-$(SYSTEM).o power.o
-
 promote:
 
 include $(BASEDIR)/makefiles/Makefile.common
 
+power.o: power-$(SYSTEM).o
+	@cp power-$(SYSTEM).o power.o
+
 arch: $(ARCH).o
-- 
